erica/api/v2/responses/model.py

```diff
@@ -54,6 +57,9 @@ class FreischaltcodeRequestAndActivationResponseDto(ResponseGetFromQueue):
 
 class FreischaltcodeRevocationResponseDto(ResponseGetFromQueue):
     result: Optional[TransferTicketAndIdnr]
+    
+class GrundsteuerResponseDto(ResponseGetFromQueue):
+    result: Optional[ResultGrundsteuerFromQueue]
```

Comment:
The result is the same as the one we have for the send est right? Why not using the same class? Maybe renaming it to be more general?

Reply:
Done